assets/css/item.css

```diff
@@ -19,4 +19,11 @@ body {background-color: #f1f2f6;}
   .item {
     width : 16.66666667%;
   }
+}
+
+p.desc {
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp:1;
+  display: -webkit-box;
```

Comment:
IE does not support `line-clamp`.
http://caniuse.com/#feat=css-line-clamp

OK. I will workaround codes